app/src/main/AndroidManifest.xml

```diff
@@ -157,7 +157,8 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.topic.revisioncard.RevisionCardActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:label="@string/revision_card_activity_title"/>
```

Comment:
This requires reformatting. Use `ctrl+alt+L` to reformat.

Reply:
When I try to reformat the file, it changes a lot of lines should I do or leave it.

Reply:
here we only need to add a space to the last line, i.e we need to change the last line to `android:label="@string/revision_card_activity_title" />`

Reply:
@rt4914 will it be okay to format the entire AndroidManifest.xml here as it changes a lot of unrelated code?

Reply:
@MaskedCarrot I made the change you told me to but I squashed the commits on my local machine and now it's rejecting when I'm trying to send it. Can you help me?